.github/workflows/ci-java.yml

```diff
@@ -18,7 +18,7 @@ jobs:
           fetch-depth: 2
       - id: workflow_files
         run: |
-          echo "::set-output name=files::$(git diff --name-only --diff-filter=ACMRT HEAD^ HEAD | grep -E '${{ env.workflow_files_re }}')"
+          echo "::set-output name=files::$(git diff --name-only --diff-filter=ACMRTD HEAD^ HEAD | grep -E '${{ env.workflow_files_re }}')"
```

Comment:
As discussed, I think the on condition here should probably be:
```
on:
  pull_request:
    paths:
      - apps/**
```

Reply:
Please see https://github.com/CMSgov/beneficiary-fhir-data/pull/773